build.rs

```diff
@@ -121,7 +121,7 @@ fn configure_libm(target: &Target) {
     }
 
     println!("cargo:rustc-check-cfg=cfg(optimizations_enabled)");
-    if target.opt_level >= 2 {
+    if !matches!(target.opt_level.as_str(), "0" | "1") {
```

Comment:
is "z" optimizations_enabled?

Reply:
For the purposes of this it is fine, the config is only used in `libm`'s test-related code. I only included it here for consistency if we wind up doing something similar in `compiler-builtins/testcrate`.